custom-script/utils.py

```diff
@@ -124,25 +146,59 @@ def predict_on_sh(model_script, bbox, size, timestamp, config):
     return request.get_data()[0]
 
 
-def get_predictions(patch, model, config):
-    model_script = parse_model(model, None)
-    sh_prediction = predict_on_sh(model_script, patch.bbox, (64, 64), patch.timestamp[0], config)
-
+def get_model_predictions(patch: EOPatch, model: Any) -> np.ndarray:
     features = patch.data["FEATURES"][0]
-    f_s = features.shape
-    model_prediction = model.predict_proba(features.reshape(f_s[0] * f_s[1], f_s[2]))[..., -1].reshape(f_s[0], f_s[1])
+    height, width, nfeats = features.shape
+    reshaped_features = features.reshape(height * width, nfeats)
+    return model.predict_proba(reshaped_features)[..., -1].reshape(height, width)
 
-    return features[..., [2, 1, 000]], sh_prediction, model_prediction
 
+def get_sh_predictions(patch: EOPatch, model: Any, config: SHConfig) -> np.ndarray:
+    model_script = parse_model(model, None)
+    return predict_on_sh(model_script, patch.bbox, (64, 64), patch.timestamp[0], config)
+
+
+def print_results(
+    f1_scores: np.ndarray,
+    recall: np.ndarray,
+    precision: np.ndarray,
+    predict_labels_test: np.ndarray,
+    labels_test: np.ndarray,
+):
+    class_names = ["non-water", "water"]
+
+    print("Classification accuracy {:.1f}%".format(100 * metrics.accuracy_score(labels_test, predict_labels_test)))
```

Comment:
use f-strings instead of formats. Pretty sure one of the linters should've marked that as a possible improvement

Reply:
done